src/cozmo/objects.py

```diff
@@ -537,19 +542,145 @@ def marker_height_mm(self):
     #### Commands ####
 
 
+class _CustomObjectType(collections.namedtuple('_CustomObjectType', 'name id')):
+    # Tuple mapping between CLAD ActionResult name and ID
+    # All instances will be members of ActionResults
+
+    # Keep _ActionResult as lightweight as a normal namedtuple
+    __slots__ = ()
+
+    def __str__(self):
+        return 'CustomObjectTypes.%s' % self.name
+
+
 class CustomObjectTypes:
-    '''Defines all available object types.
+    '''Defines all available custom object types.
+
+    For use with world.define_custom methods such as
+    :meth:`cozmo.world.World.define_custom_box`,
+    :meth:`cozmo.world.World.define_custom_cube`, and
+    :meth:`cozmo.world.World.define_custom_wall`
+    '''
+
+    #: CustomType00 - the first custom object type
+    CustomType00 = _CustomObjectType("CustomType00", _clad_to_engine_cozmo.ObjectType.CustomType00)
+
+    #:
```

Comment:
With a `#:` in front of it they're listed in the documentation, otherwise there's no way to see them. I excluded a text description for each (just one for the 1st and last) as that kept that generated documentation looking cleaner.